css/cssom-view/matchMedia.xht

```diff
@@ -1,188 +0,0 @@
-
-            test(function(){
-                assert_false(mql.matches);
-            }, "MediaQueryList.matches for \"not all\"");
-
-            test(function(){
-                mql = iframe_window.matchMedia("(max-width: 199px), all and (min-width: 200px)");
-                assert_equals(mql.media, "(max-width: 199px), (min-width: 200px)");
-                assert_true(mql.matches);
-            }, "MediaQueryList.matches for \"(max-width: 199px), all and (min-width: 200px)\"")
-
-            test(function(){
-                mql = iframe_window.matchMedia("(min-aspect-ratio: 1/1)");
-                assert_true(mql.matches);
-            }, "MediaQueryList.matches for \"(min-aspect-ratio: 1/1)\"");
-
-            test(function(){
-                mql = iframe_window.matchMedia("(width: 200px)");
-                assert_true(mql.matches);
-            }, "MediaQueryList.matches for \"(width: 200px)\"");
-
-            test(function(){
-                mql1 = iframe_window.matchMedia("(max-height: 50px)");
-                assert_false(mql1.matches);
-            }, "MediaQueryList.matches for \"(max-height: 50px)\"");
-
-            test(function(){
-                mql2 = iframe_window.matchMedia("(min-width: 150px)");
-                assert_true(mql2.matches);
-            }, "MediaQueryList.matches for \"(min-width: 150px)\"");
-
-            var resizeTest = async_test("Resize iframe from 200x100 to 200x50, then to 100x50");
-            var listenerOrderTest = async_test("Listeners are called in the order which they have been added");
-            var duplicateListenerTest = async_test("Listener added twice is only called once.");
-
-            window.onload = function(){
-
-                var rmListener = function(x){
-                    resizeTest.step(function(){
-                        assert_unreached("removeListener was not successful.");
-                    });
-                };
-
-                var dupListener = function(x){
-                    duplicateListenerTest.step(function(){
-                        assert_false(mql1.dupListenerCalled, "Check that this listener has not been called before.");
-                        mql1.dupListenerCalled = true;
-                    });
-                };
-
-                mql1.firstListenerCalled = false;
-                mql1.dupListenerCalled = false;
-                // Add listener twice and remove it below. Should not be called.
-                mql1.addListener(rmListener);
-                mql1.addListener(rmListener);
-                // Add listener twice. Should only be called once.
-                mql1.addListener(dupListener);
-                mql1.addListener(dupListener);
-
-                mql1.addListener(function(x){
-                    resizeTest.step(function(){
-                        assert_equals(x, mql1, "Check that the MediaQueryList passed to the handler is the same that addListener was invoked on.");
```

Comment:
This assertion was outdated (failing in all browsers except Safari): listeners are now called with `MediaQueryListEvent` rather than `MediaQueryList` itself.